s3api/controllers/base.go

```diff
@@ -3099,7 +3140,9 @@ func (c S3ApiController) DeleteActions(ctx *fiber.Ctx) error {
 	// The AWS CLI sends 'True', while Go SDK sends 'true'
 	bypass := strings.EqualFold(bypassHdr, "true")
 
-	err = auth.CheckObjectAccess(ctx.Context(), bucket, acct.Access, []types.ObjectIdentifier{{Key: &key, VersionId: &versionId}}, bypass, c.be)
+	fmt.Println("checking object access: ", bypass, IsBucketPublic)
```

Comment:
debugging logging removal missed? fine to leave in if useful but move to debug only.